gradle/docker.gradle

```diff
@@ -42,7 +42,7 @@ ext {
     dockerRegistry = project.hasProperty('dockerRegistry') ? dockerRegistry + '/' : ''
     dockerImageTag = project.hasProperty('dockerImageTag') ? dockerImageTag : 'latest'
     dockerImagePrefix = project.hasProperty('dockerImagePrefix') ? dockerImagePrefix : 'whisk'
-    dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 840
+    dockerTimeout = project.hasProperty('dockerTimeout') ? dockerTimeout.toInteger() : 8400
```

Comment:
why is this needed?

Reply:
I added this because my network is not good and during the installation, it times out very quick. Therefore, I increased the timeout interval.

Reply:
You can provide this value for you locally, if your network is really super bad. No need to change the default imo.

`gradle -PdockerTimeout=8400 distDocker` for example.